db/seeds.rb

```diff
@@ -64,7 +64,7 @@
 female = Animal.create!(sex: 'female')
 family = Family.create!(male: male, female: female)
 tank = Tank.create!(facility: Facility.find_by(code: 'PSRF'), name: 'AB-17')
-Operation.create!(tank: tank, animals_added: 800, animals_added_family_id: family.id, operation_date: 7.days.ago, action: 'add')
+Operation.create!(tank: tank, animals_added: 800, family: family, animals_added_family_id: family.id, operation_date: 7.days.ago, action: 'add')
```

Comment:
@mdworken  - Could you clarify what the purpose of the `animals_added_family_id` is supposed to signify? Is this leftover that can be removed?

Reply:
I think it probably is, yes!